wp-seo-main.php

```diff
@@ -72,7 +74,7 @@ function wpseo_auto_load( $class ) {
 $yoast_autoload_file = WPSEO_PATH . 'vendor/autoload.php';
 
 if ( is_readable( $yoast_autoload_file ) ) {
-	require $yoast_autoload_file;
+	$autoloader = require $yoast_autoload_file;
```

Comment:
This should be prefixed with `yoast_` as we're in the global namespace.

Reply:
```suggestion
	$yoast_autoloader = require $yoast_autoload_file;
```